src/XRP/serializer.ts

```diff
@@ -917,6 +919,16 @@ const serializer = {
     // TODO(keefertaylor): Use accountAddressToJSON() here when supported.
     return account.getValue()?.getAddress()
   },
+
+  /**
+   * Convert a Condition to a JSON representation.
+   *
+   * @param condition - The Condition to convert.
+   * @returns The Condition as JSON.
+   */
+  conditionToJSON(condition: Condition): ConditionJSON {
+    return condition.getValue_asB64()
```

Comment:
This will give you a base 64 encoded string. I think that it's meant to be encoded as hexadecimal: https://xrpl.org/escrow-object.html

You can use `Utils.toHex` and `getValue_asU8()` to get a hex encoded string. 

Aside: I think it would be okay to return a UInt8Array from this method but in other places we've opted to return a hex-encoded string so I would value consistency.

Reply:
sweet, this is the exact question i was gonna ask. will fix this